src/core/dbus-manager.c

```diff
@@ -2160,6 +2160,11 @@ static int install_error(
                                               "Unit file %s does not exist.", changes[i].path);
                         goto found;
 
+                case -EXDEV:
+                        r = sd_bus_error_setf(error, BUS_ERROR_BAD_UNIT_SETTING,
+                                              "\"%s\" is an invalid alias.", changes[i].path);
```

Comment:
hmm, I think we should say in which way "invalid".

i.e. maybe `\"%s\" unit alias doesn't match unit name` or so

Reply:
I've used the generic message on purpose.
A that point - in `dbus-manager.c` - it is hard to say more; since `UnitFileChange`'s instance is carrying an errno only.
It can be: _invalid unit name_, _source-target mismatch_ or _an incompatible suffix_ (these details should be logged by `log_full_errno`.
My point is, for the cli it is still better to see, that there is something wrong with an unit, instead of generic errno readout.